# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../pinentry/glib.patch
# Copyright (C) 2009 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur pinentry-0.7.5-orig/gtk+-2/gtksecentry.c pinentry-0.7.5/gtk+-2/gtksecentry.c
--- pinentry-0.7.5-orig/gtk+-2/gtksecentry.c	2009-02-17 14:36:11.000000000 +0000
+++ pinentry-0.7.5/gtk+-2/gtksecentry.c	2009-02-17 14:38:03.000000000 +0000
@@ -270,7 +270,7 @@
 
 
 gpointer
-g_malloc(gulong size)
+g_malloc(gsize size)
 {
     gpointer p;
 
@@ -288,7 +288,7 @@
 }
 
 gpointer
-g_malloc0(gulong size)
+g_malloc0(gsize size)
 {
     gpointer p;
 
@@ -308,7 +308,7 @@
 }
 
 gpointer
-g_realloc(gpointer mem, gulong size)
+g_realloc(gpointer mem, gsize size)
 {
     gpointer p;
 
